pkg/cloud/scope/clients.go

```diff
@@ -270,9 +359,10 @@ func getUserAgentHandler() request.NamedHandler {
 
 // AWSClients contains all the aws clients used by the scopes.
 type AWSClients struct {
-	ASG             autoscaling.Client
-	EC2             ec2iface.EC2API
-	ELB             elbiface.ELBAPI
-	SecretsManager  secretsmanageriface.SecretsManagerAPI
-	ResourceTagging resourcegroupstaggingapiiface.ResourceGroupsTaggingAPIAPI
+	ASG               autoscaling.Client
+	EC2               ec2iface.EC2API
+	ELB               elbiface.ELBAPI
```

Comment:
Should we not be adding the ELBv2 interface here?

Reply:
It is not used anywhere in the updated code as far as I can recall, and to keep changes slim I didn’t add it here. Do you think this needs to be a part of the PR?